### PR TITLE
i18n(zh-cn): update `environment-variables.mdx`

### PR DESCRIPTION
**Pull request by imbant**
#### What kind of changes does this PR include?

- Translated content

#### Description

From [translation tracking system](https://i18n.docs.astro.build/) I found this page needs updating.The [source change history](https://github.com/withastro/docs/commits/main/src/content/docs/en/guides/environment-variables.mdx?since=2023-04-14T18:19:31.000Z) shows a new commit: `Explain how to get environment variables in SSR mode`, so I translated it.

### CODE REVIEW

**Comment by imbant** [comment]
I noticed there is a link to a missing Chinese page (/zh-cn/guides/integrations-guide/cloudflare#环境变量) in my commit. Maybe should merge my PR after this [PR](https://github.com/withastro/docs/pull/4090) is merged

File: src/content/docs/zh-cn/guides/environment-variables.mdx [hunk 1]
```diff
@@ -91,6 +92,9 @@ const data = fetch(`${import.meta.env.PUBLIC_POKEAPI}/pokemon/squirtle`);
 因为 Vite 静态地替换了 `import.meta.env`，所以你不能用 `import.meta.env[key]` 这样的动态键来访问它。
 :::
 
+使用 SSR 时，可以根据所使用的 SSR 适配器，在运行时访问环境变量。对于大部分适配器，可以通过 `process.env` 访问环境变量，而 Deno 适配器则需要使用 `Deno.env.get()`。Cloudflare 有它[自己的方式](/zh-cn/guides/integrations-guide/cloudflare#环境变量) 来处理环境变量。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/environment-variables.mdx:95]:
```suggestion
使用 SSR 时，可以根据所使用的 SSR 适配器，在运行时访问环境变量。对于大部分适配器，可以通过 `process.env` 访问环境变量，而 Deno 适配器则需要使用 `Deno.env.get()`。Cloudflare 有它[自己的方式](/zh-cn/guides/integrations-guide/cloudflare/#环境变量) 来处理环境变量。
```

Comment (imbant) [reply src/content/docs/zh-cn/guides/environment-variables.mdx:95]: Thanks for checking!

Comment (liruifengv) [reply src/content/docs/zh-cn/guides/environment-variables.mdx:95]: You can use `commit suggestion`

**Review by liruifengv — APPROVED** [review]
LGTM

**Pull request merged**